### DISCUSSION TITLE
Solid and Duotone are combined now?

### DISCUSSION THREAD

**Discussion by MarkoOleksiyenko** [FortAwesome/Font-Awesome#20301] [General]
Hello guys, 

Noticed that after 6.6.0 in the styles there is only Solid, Regular, Light and Thin but no duotone.

The duotone icons have also the fa-solid class that makes no sense.

Can you please tell me whether it is an intended change?

Best regards,
Marko

**Comment by sensibleworld** [comment]
@MarkoOleksiyenko Sorry for any confusion. Since its inception, Duotone has always been a family (not a style). With v6.6, we've just updated a few things so that we're consistently referring to it as such. All of the old class naming should continue to work, however.

So the families that exist on Font Awesome are:
fa-classic
fa-sharp
fa-duotone
fa-sharp-duotone

And each of those families has styles included. For now, we only have the `solid` style for Duotone and Sharp Duotone, but that may change in the future.

**Reply by MarkoOleksiyenko** [reply]
Thanks for the answer @sensibleworld ! It is clear now for me 👍
